### ISSUE TITLE
Editable custom attributes

### ISSUE THREAD

**Issue by Mracobes9** [mattermost-community/mattermost-plugin-custom-attributes#12]
Hi. Can i add attributes which users can edit in profile settings ? How ?

**Comment by hanzei** [comment]
Changes to the webapp are needed first

**Comment by mpounsett** [comment]
@hanzei Are there any indications of when we might see those changes?  I have a MM server with people from many different organizations on it, and I'd really like to give them an "Organization" profile field (in addition to Position) that they can populate.  I'm sure there are other use cases for user-editable profile attributes as well; I can imagine hobbiest servers wanting to add a list of interests, for example.

**Comment by fergusoreilly** [comment]
Adding a use case example here: would like to have user-editable custom profile attributes to allow users to add their mobile phone number to their profile.

Ideally when creating a new custom attribute, the admin should be able to specify help text to guide the user when filling out the profile attribute, like the Mattermost UX does for the default profile attributes:

<img width="595" alt="profile nickname help text example" src="https://user-images.githubusercontent.com/97966619/171982278-448087cc-cf76-4c8f-b54b-45d70ffab8d2.png">

**Comment by erfantkerfan** [comment]
+1
This is exactly what my team needs.
I would appreciate it if this would be added to the capabilities of this plugin.

**Comment by kelsen** [comment]
Any chance this would be implemented?

**Comment by mpounsett** [comment]
@hanzei : You added the `needs-spec` tag to this issue.  Can you outline (or point to documentation) what details you'd like to see?  I'm happy to write something up if it moves this along.

**Comment by hanzei** [comment]
Actually, custom, editable attributes have just been shipped via v10.5 of Mattermost. Please check out https://forum.mattermost.com/t/design-preview-custom-profile-attributes/18998 for a preview of the interface and https://docs.mattermost.com/about/mattermost-v10-changelog.html#id32 for more details.

**Comment by kelsen** [comment]
this is an enterprise-only feature.

**Comment by mpounsett** [comment]
Yes, being enterprise-only it doesn't really help those of us with large pro servers.

**Comment by dadatuputi** [comment]
> Actually, custom, editable attributes have just been shipped via v10.5 of Mattermost. Please check out https://forum.mattermost.com/t/design-preview-custom-profile-attributes/18998 for a preview of the interface and https://docs.mattermost.com/about/mattermost-v10-changelog.html#id32 for more details.

@hanzei Understood if you don't want to implement this, but the solution you linked to is enterprise only. This feature request would make this plugin extremely useful.
